gammapy/image/models/core.py

```diff
@@ -322,7 +322,7 @@ class SkyEllipse(SkySpatialModel):
         from gammapy.image.models.core import SkyEllipse
         from gammapy.maps import Map, WcsGeom
 
-        model = SkyEllipse("2 deg", "2 deg", "1 deg", 0.8, "30 deg")
+        model = SkyEllipse("2 deg", "2 deg", "1 deg", 0.8, "20 deg", frame='galactic')
```

Comment:
I guess you want to put `"galactic"` rather than `'galactic'`, to be compliant with black.